vocabularies/Common.xml

```diff
@@ -417,17 +418,17 @@ Example for dynamic use: in a travel expense report the property `DestinationCou
           <Annotation Term="Core.Description" String="..." />
         </Property>
       </ComplexType>
-      <Term Name="Timestamp" Type="Edm.DateTimeOffset">
+      <Term Name="Timestamp" Type="Edm.DateTimeOffset" Nullable="false">
         <Annotation Term="Common.Experimental" />
         <Annotation Term="Core.Description" String="..." />
         <Annotation Term="Common.IsInstanceAnnotation" />
       </Term>
```

Comment:
Aren't instance annotations nullable by nature, because you can never assume the server generates them?

Reply:
Absence of an instance annotation means it is `undefined`, which is different from `null`.